src/6_branch.js

```diff
@@ -1466,7 +1467,7 @@ Branch.prototype['addListener'] = function(event, listener) {
 
 /** =WEB
  * @function Branch.removeListener
- * @param {function(String)} listener - _required_ - Reference to the listening function you
+ * @param {function(String, Object)} listener - _required_ - Reference to the listening function you
```

Comment:
let's remove the additional Object param as the definition for removeListener has not changed